tests/drivers/spi/spi_loopback/boards/mimxrt1170_evk_cm7.conf

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_SPI_MCUX_LPSPI_DMA=y
```

Comment:
need to add to the testcase.yaml to test all combinations of DMA and ASYNC kconfigs

Reply:
Please check this.

Reply:
I have added another testcase.